### ISSUE TITLE
Cannot access local-data-api on AWS CodeBuild

### ISSUE THREAD

**Issue by kobanyan** [koxudaxi/local-data-api#121]
**Describe the bug**

With `v0.6.0` and above, I get the below error at trying to access local-data-api on AWS CodeBuild.

```
[Container] 2021/04/14 05:39:38 Running command curl -i "http://localhost:8080"
```

**To Reproduce**

1. Create a stack

```yaml
AWSTemplateFormatVersion: '2010-09-09'
Resources:
  CodeBuildProject:
    Type: AWS::CodeBuild::Project
    Properties:
      Name: local-data-api
      Source:
        Location: 'https://github.com/koxudaxi/local-data-api'
        Type: GITHUB
        BuildSpec: |
          version: 0.2
          phases:
            build:
              commands:
                - docker-compose -f docker-compose-postgres.yml up -d
                - sleep 30s
                - curl -i "http://localhost:8080"
      Environment:
        Type: LINUX_CONTAINER
        Image: aws/codebuild/standard:5.0
        ComputeType: BUILD_GENERAL1_SMALL
        PrivilegedMode: true
      TimeoutInMinutes: 60
      QueuedTimeoutInMinutes: 60
      ServiceRole: !GetAtt IAMRoleCodeBuildServiceRole.Arn
      Artifacts:
        Type: NO_ARTIFACTS
  IAMRoleCodeBuildServiceRole:
    Type: AWS::IAM::Role
    Properties:
      RoleName: 'codebuild-local-data-api-service-role'
      AssumeRolePolicyDocument:
        Version: '2012-10-17'
        Statement:
          - Action:
              - sts:AssumeRole
            Effect: Allow
            Principal:
              Service:
                - codebuild.amazonaws.com
      ManagedPolicyArns:
        - 'arn:aws:iam::aws:policy/AdministratorAccess'
```

2. Start build

**Expected behavior**

Successful with `v0.5.8`

```
[Container] 2021/04/14 03:50:22 Running command curl -i "http://localhost:8080"
HTTP/1.1 404 Not Found
date: Wed, 14 Apr 2021 03:50:24 GMT
server: uvicorn
content-length: 22
content-type: application/json
 
{"detail":"Not Found"}
```

**Comment by koxudaxi** [comment]
@kobanyan 
Thank you for creating this issue.
I can reproduce the problem.
I have released a fixed version as `0.6.8`

Thank you very much.

**Comment by kobanyan** [comment]
Thanks for the quick fix!
